### ISSUE TITLE
Bad results for Places2 with rect mask

### ISSUE THREAD

**Issue by silence1114** [shepnerd/inpainting_gmcnn#23]
Hello, I find that the pre-trained model given "places2_512x680_freeform" works well with stroke mask, but the results are poor with rect mask. Can you provide the pre-trained model with rect mask? Thanks.

**Comment by shepnerd** [comment]
We may release that model in the future. Currently, you can just train model with rect masks using the pretrained model.

**Comment by silence1114** [comment]
> We may release that model in the future. Currently, you can just train model with rect masks using the pretrained model.

If I train the model with rect masks using the pre-trained model, is the parameter setting consistent with the one mentioned in #9 ?

**Comment by shepnerd** [comment]
That places2-stroke model is trained on 256x256 images from places2 dataset with g_cnum=32. It'd better continue to train it in the same resolution. Also, the results of our method are unstable on the large-scale dataset like places2.

**Comment by ljjcoder** [comment]
@shepnerd I train the places2-rect model from the places2-stroke model you give with pretrain=0, and I have trained 5days. Some result like this:
![Places365_val_00000020](https://user-images.githubusercontent.com/40750189/60933769-4f717180-a2f6-11e9-991c-8fa7b9426f44.png)
![Places365_val_00000018](https://user-images.githubusercontent.com/40750189/60933778-56987f80-a2f6-11e9-965d-73569c67c7f8.png)
and  I test the psnr to compare with CA(Generative Image Inpainting with Contextual Attention) on places2 with 1000 images random select from val dataset. The psnr of CA and the trained rect model are 18.158, 17.900. It is seems  that the rect model is no good as CA.  Do you have any advise to improve result of rect model？

**Comment by shepnerd** [comment]
For this generation task, psnr is just for reference. If you just wanna
improve the psnr of final results, you can just increase the weight of the
reconstruction loss and decrease others relatively. You can even change the
l1 norm in the reconstruction loss to l2 norm.

ljjcoder <notifications@github.com> 于2019年7月10日周三 上午9:50写道：
